Sming/SmingCore/Network/HttpRequest.cpp

```diff
@@ -136,6 +136,11 @@ HttpParseResult HttpRequest::parseHeader(HttpServer *server, pbuf* buf)
 		line = nextLine + 2;
 	} while(nextLine != -1);
 
+	if (getContentLength() > 0)
+	{
+		content = NetUtils::pbufStrCopy(buf, headerEnd + 4, getContentLength());
```

Comment:
We can't be sure what we will have full response text at this place. It can be received later in next tcp packet.